bindings/python/utils/pickle-map.hpp

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
The codes of PickleMap and PickleVector are exactly the same. It would be worthy to avoid this duplication.
Can you just make PickleMap inherited from PickleVector for instance?

Reply:
Done.